publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -487,7 +501,7 @@ public void shutdownCompleted(ShutdownSignalException cause) {
         } catch (Exception e) {
             log.error(e.getMessage(), e);
             if(!channel.isOpen()&& rabbitConnection.isOpen()){
-                throw new RemRemPublishException("Channel was closed for Rabbitmq connection ::" + factory.getHost() + factory.getPort());
+                throw new RemRemPublishException("Channel was closed for Rabbitmq connection ::" + factory.getHost() + factory.getPort() + e.getMessage());
```

Comment:
Pls update ALL occasions in the class to have the same format of getHost + getPort (what I commented earlier, host and port separated with a ':', they really must be separated for readability), like:

factory.getHost() + ":" + factory.getPort()

Reply:
I suggest to create a new constructor
  RemRemPublishException(String message, ...Factory factory, Exception e) {
   ...
}
In that way we can have message creation at one place.